README.md

```diff
@@ -2,13 +2,13 @@
 
 <table style="width: 100%; border-style: none;"><tr>
 <td width="140px" style="text-align: center;"><img src="apple_default.png" style="max-width:100%" /></td>
-<td><strong>Visual Studio Team Services Extension for the Apple App Store</strong><br />
+<td><strong>Azure DevOps Extension for the Apple App Store</strong><br />
 <i>Provides build/release tasks that enable performing continuous delivery to Apple's App Store from an automated VSTS build or release definition</i><br />
```

Comment:
Azure DevOps Pipeline

Reply:
Extension is for Azure DevOps, it just happens to have only pipeline tasks right now.